pages/index.html

```diff
@@ -4,9 +4,31 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>HOME</title>
+  <link href="../stylesheets/style.css" rel="stylesheet">
+  <link href="../stylesheets/index.css" rel="stylesheet">
 </head>
+
 <body>
   
+  <header>
+    <h1>DOMINIQUE DABA</h1>
+  </header>
+
+  <section>
```

Comment:
Great job using semantic HTML! I would recommend using the `<main>` tag here instead.